spec/operators/switchMapFirst-spec.js

```diff
@@ -33,6 +33,31 @@ describe('Observable.prototype.switchMapFirst()', function () {
     expectSubscriptions(x.subscriptions).toBe(xsubs);
   });
 
+  it('should raise error if project throws', function () {
+    var x = cold(     '--a--b--c--|                              ');
+    var e1 =   hot('---x---------y-----------------z-------------|');
+    var expected = '---#';
```

Comment:
Good stuff, but can you add also an unsubscription assertion here for the error raised in this test case?